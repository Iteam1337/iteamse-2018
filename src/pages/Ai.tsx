import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import Helmet from 'react-helmet'
import { AiPageQuery } from '../../typings/iteamse'
import AiTable from '../components/Blocks/AiTable'
import Block from '../components/Blocks/Block'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import queryWithLoading from '../components/QueryWithLoading/QueryWithLoading'
import Team from '../components/Team/Team'

export const AI_PAGE_QUERY = gql`
  query AiPage {
    pageAi {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      team
      aboutTitle
      aboutText
      contentImage
      offersText
      offersTitle
      contactTitle
    }
  }
`

class AiQuery extends Query<AiPageQuery> { }
const AiQueryWithLoading = queryWithLoading(AiQuery)

export class Ai extends React.Component {
  render() {
    return (
      <AiQueryWithLoading query={AI_PAGE_QUERY}>
        {({ pageAi }: { pageAi: any }) => {
          return (
            <>
              <Helmet>
                <title>Iteam | AI</title>
                <meta property="og:title" content="Iteam | AI" />
                <meta property="twitter:title" content="Iteam | AI" />
                {pageAi.headerImage && (
                  <meta
                    property="og:image"
                    content={`https:${pageAi.headerImage}`}
                  />
                )}
              </Helmet>
              <Header
                backgroundImage={pageAi.headerImage}
                messageBgColor={pageAi.headerTextBgColor}
                messageOne={pageAi.headerText1}
                messageTwo={pageAi.headerText2}
              />

              <GridColumn>
                <Block title={pageAi.aboutTitle}>{pageAi.aboutText}</Block>

                <AiTable title={pageAi.offersTitle} body={pageAi.offersText} />
              </GridColumn>

              <Team
                bgColor="red"
                shortName={pageAi.team}
                callToAction={pageAi.contactTitle}
              />
            </>
          )
        }}
      </AiQueryWithLoading>
    )
  }
}

export default Ai
