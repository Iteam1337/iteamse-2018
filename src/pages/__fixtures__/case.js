export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
