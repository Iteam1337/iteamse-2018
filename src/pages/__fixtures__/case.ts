export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
