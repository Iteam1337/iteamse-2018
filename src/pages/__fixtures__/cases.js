export const cases = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'vimla',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/68OlBclfQAGMcEUsQgcIUY/9bdf52bb8a9a17be06e11ef3fa32d3de/vimla-case-phone.png',
    title: 'Vimla',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'tsab',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/UGucAeFycSqIGw0CAc0ys/3d58300261bcff64dba751de9228a150/tsab-case-phone.png',
    title: 'Taxi Stockholm',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'seb',
    thumbnailImage: null,
    title: 'SEB',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'arbetsformedlingen',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
    title: 'Arbetsförmedlingen',
    __typename: 'Case',
  },
]
