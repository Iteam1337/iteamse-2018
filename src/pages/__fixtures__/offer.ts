// tslint:disable:object-literal-sort-keys
export const offers = [
  {
    offerIntroImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5n7J4mzx3UWam0YwgsUQm/8e00d12d6e80eba927b2a313fb250d87/1.png',
    offerTitle: 'Kod',
    offerIntroText:
      'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med. ',
    offerLead:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. ',
    offerUspOneTitle: 'Utveckling av pilot',
    offerUspOneText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspTwoTitle: 'API',
    offerUspTwoText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspThreeTitle: 'Mobila applikationer',
    offerUspThreeText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspFourTitle: 'AI-lösningar',
    offerUspFourText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi\n',
    offerIllustrationImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5SAhc8YuL6OaQQsC6miokK/7c09d594e44ffa47781bf1bdb21aae7f/illustrations-group-2.png',
    offerOrder: 1,
    __typename: 'Offer',
  },
]
