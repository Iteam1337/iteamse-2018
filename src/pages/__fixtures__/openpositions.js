export const openpositions = [
  {
    id: 'junior-ux-designer',
    location: 'Stockholm',
    role:
      'Som UX designer på Iteam är din roll att hjälpa våra kunder med digitala lösningar som skapar värde på riktigt. Som gör något bra både för kunden, slutanvändaren och samhället. Med hjälp av olika researchmetoder undersöker du både användarens och kundens behov och kontext, samt definierar och testar användarupplevelser.\n\nDu delar med dig av din kunskap inom user experience. Du förstår också att om lösningar ska vara verkligt användarvänliga behöver även arbetsprocesserna och kulturen vara användarcentrerad. Vi strävar mot att UX aldrig ska få vara bara en fas eller en eftertanke. Det ska genomsyra hela projektet. Alltid. ',
    title: 'Junior UX Designer',
    __typename: 'OpenPosition',
  },
]
