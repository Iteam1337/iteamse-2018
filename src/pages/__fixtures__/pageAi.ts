export const pageAi = {
  __typename: 'PageAi',
  aboutText:
    'We at Iteam have the knowledge and competence to help you build data products. Not only that, we can help you use the data in your data products to make better decisions at different layers of your organizations: from day-to-day decisions to staratic decisions.\n\nWe have seen enormous efforts and progress in AI in sectors such as finance and banking, transport, health, telecom and education and we are ready to help you model events and data you need to be faster in this constantly changing market of digital world.',
  aboutTitle: 'AI',
  contactTitle: 'Contact us!',
  contentImage: null,
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/79Qpsnr42s0AUyeEqKeOgI/69413d924ff24471c484e43c3577d576/Amir-Code-Lunch.jpg',
  headerText1: 'Våra erbjudanden inom AI',
  headerText2: null,
  headerTextBgColor: '#5DF9AE',
  linkText: 'Läs mer om vårt AI-erbjudande',
  offersText:
    'We will always be open to your need, but sometimes clients are more comfortable with having services packaged in terms of hours and workload. We have three packages ready for your needs as you can see below:',
  offersTitle: 'Vad vi kan erbjuda',
  shortDescription:
    'We are offering an Artificial Intelligence package to help you build AI products and services. We have a very competent Data Science team that will help you grow in the growing market of Artificial Intelligence.',
  team: ['ara'],
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/1oqMfeZBnuQAAGweIGCKEW/60143ba037ba021f0f1090f571207e8a/Laptop-Large.png',
}
