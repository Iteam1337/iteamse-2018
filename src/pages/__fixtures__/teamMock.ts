import { TEAM_QUERY } from '../../components/Team/Team'
import { teamMembers } from './teamMembers'

export const TeamQueryMock = {
  request: {
    query: TEAM_QUERY,
  },
  result: {
    data: {
      teamMembers,
    },
  },
}
