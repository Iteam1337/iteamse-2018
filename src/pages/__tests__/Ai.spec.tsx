import React from 'react'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageAi } from '../__fixtures__/pageAi'
import { TeamQueryMock } from '../__fixtures__/teamMock'
import Ai, { AI_PAGE_QUERY } from '../Ai'

const mocks = [
  {
    request: {
      query: AI_PAGE_QUERY,
    },
    result: {
      data: {
        pageAi,
      },
    },
  },
  TeamQueryMock,
]

describe('components/Ai', () => {
  it('renders Ai', async () => {
    const { container } = render(
      <MockedQuery mocks={mocks}>
        <Ai />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
