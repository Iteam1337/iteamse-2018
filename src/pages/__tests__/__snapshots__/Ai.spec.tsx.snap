// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Ai renders Ai 1`] = `
<div>
  <div>
    <div
      class="sc-lhVmIH fTQKsL"
    >
      <div
        class="sc-bYSBpT bdgTKs"
      >
        <div
          class="sc-kafWEX doqriu"
        >
          <a
            class="sc-feJyhm bzbyYc"
          >
            <img
              class="sc-iELTvK kXWcVM"
              data-testid="logo"
              path="test-file-stub"
            />
          </a>
          <div
            class="sc-cmTdod gUTeYg"
          >
            <a
              class="sc-btzYZH kmRwNW"
              aria-current="false"
            >
              Case
            </a>
            <a
              class="sc-btzYZH kmRwNW"
              aria-current="false"
            >
              AI
            </a>
            <a
              class="sc-btzYZH kmRwNW"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-btzYZH kmRwNW"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-btzYZH kmRwNW"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-btzYZH kmRwNW"
              aria-current="false"
            >
              Om
            </a>
            <a
              class="sc-btzYZH kmRwNW"
              aria-current="false"
            >
              Drift & Support
            </a>
            <div
              class="sc-jwKygS fTIVRr"
            />
          </div>
        </div>
        <div
          class="sc-elJkPf kqIJd"
        >
          <h1
            class="sc-jtRfpW cAjlcN"
          >
            <span>
              Våra erbjudanden inom AI
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sc-iwsKbI fwFOwW"
  >
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              AI
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  We at Iteam have the knowledge and competence to help you build data products. Not only that, we can help you use the data in your data products to make better decisions at different layers of your organizations: from day-to-day decisions to staratic decisions.
                </p>
                <p>
                  We have seen enormous efforts and progress in AI in sectors such as finance and banking, transport, health, telecom and education and we are ready to help you model events and data you need to be faster in this constantly changing market of digital world.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="table-"
      >
        <div
          class="sc-cSHVUG hCuviw"
        >
          <div
            class="sc-kAzzGY jHprQs"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Vad vi kan erbjuda
            </h3>
          </div>
          <div
            class="sc-chPdSV mxuDG"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  We will always be open to your need, but sometimes clients are more comfortable with having services packaged in terms of hours and workload. We have three packages ready for your needs as you can see below:
                </p>
              </div>
            </div>
            <div
              class="ReactTable"
            >
              <div
                class="rt-table"
                role="grid"
              >
                <div
                  class="rt-thead -header"
                  style="min-width: 400px;"
                >
                  <div
                    class="rt-tr"
                    role="row"
                  >
                    <div
                      class="rt-th"
                      role="columnheader"
                      tabindex="-1"
                      style="width: 100px;"
                    >
                      <div
                        class=""
                      />
                    </div>
                    <div
                      class="rt-th"
                      role="columnheader"
                      tabindex="-1"
                      style="width: 100px;"
                    >
                      <div
                        class=""
                      >
                        Package 1
                      </div>
                    </div>
                    <div
                      class="rt-th"
                      role="columnheader"
                      tabindex="-1"
                      style="width: 100px;"
                    >
                      <div
                        class=""
                      >
                        Package 2
                      </div>
                    </div>
                    <div
                      class="rt-th"
                      role="columnheader"
                      tabindex="-1"
                      style="width: 100px;"
                    >
                      <div
                        class=""
                      >
                        Package 3
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="rt-tbody"
                  style="min-width: 400px;"
                >
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -odd"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          FTE
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        4
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        4
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        4
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -even"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Weeks
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        6
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        9
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        12
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -odd"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Data Scientist
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -even"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          DevOps Engineer
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        0.5
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        0.5
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -odd"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Data Engineer
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1.5
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1.5
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -even"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Developer
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -odd"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Hours
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        960
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        1440
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        2160
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -even"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Deliverables
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      />
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      />
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      />
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -odd"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Datasource
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        Database Export
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        Live Data Streams
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        Live Data Streams
                      </div>
                    </div>
                  </div>
                  <div
                    class="rt-tr-group"
                    role="rowgroup"
                  >
                    <div
                      class="rt-tr -even"
                      role="row"
                    >
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        <div
                          class="sc-kgoBCf bnKAYi"
                        >
                          Model
                        </div>
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        Proof-of-Concept Model
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        Proof-of-Concept Model
                      </div>
                      <div
                        class="rt-td"
                        role="gridcell"
                        style="width: 100px;"
                      >
                        Production Model
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="-loading"
              >
                <div
                  class="-loading-inner"
                >
                  Loading...
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
